docs/glossary.md

```diff
@@ -302,10 +320,73 @@ _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
 #### [Namespace](/namespaces)
 
-A Namespace is a unit of isolation within the Temporal Platform
+A Namespace is a unit of isolation within the Temporal Platform.
 
 _Tags: [term](/tags/term), [explanation](/tags/explanation)_
 
+#### [Nexus API Definitions](/nexus#nexus-api-defintions)
+
+A common code package, schema, or documentation that a Caller can use to obtain Service and Operation names as associated input/output types a Service will accept for a given Operation.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus API Registry](/nexus#nexus-api-registry)
+
+An account-scoped list of Endpoints and Services available for use within a Temporal Cloud account, including an Endpoint description of the Services and Operations which the Endpoint supports. 
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Client](/nexus#nexus-client)
+
+Used by the Caller to connect to a Nexus Endpoint and Nexus Service.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Endpoint](/nexus#nexus-endpoint)
+
+A Nexus Endpoint is like an API proxy that can serve one or more Nexus Services from a target such as a Worker listening on a target Temporal Namespace and target Task Queue. 
+This allows service providers to present a clean API contract and hide the underlying implementation, which may consist of many internal Workflows. 
+Multiple Nexus Endpoints can target the same Namespace, and over time a Nexus Endpoint will be able to span multiple Namespaces with service routing rules.
+
+_Tags: [term](/tags/term), [nexus](/tags/nexus), [explanation](/tags/explanation)_
+
+#### [Nexus Operation](/nexus#nexus-operation)
+
+An arbitrary-length service call that may be synchronous or asynchronous, short-lived, or long-lived, used to invoke operations across Namespace, Temporal Service, and cell boundaries and into arbitrary external systems.
```

Comment:
```suggestion
An arbitrary-duration operation that may be synchronous or asynchronous, short-lived, or long-lived, and used to connect durable executions within and across namespaces, clusters, regions, and clouds. Unlike a traditional RPC, an asynchronous Nexus Operation has an operation identity that can be used to re-attach to a long-lived Nexus Operation, for example, one backed by a Temporal Workflow. Nexus Operations support a uniform interface to get the status of an operation or its result, receive a completion callback, or cancel the operation – all of which are fully integrated into the Temporal platform.
```

Reply:
I like what you've added here. I would change "re-attach to a long-lived Nexus Operation" to "re-attach to it".